frontend/src/arkham/components/Location.vue

```diff
@@ -326,7 +345,7 @@ function onDrop(event: DragEvent) {
           v-model="showAbilities"
           :abilities="abilities"
           :frame="frame"
-          :show-move="abilities.length > 1"
```

Comment:
This is actually necessary for return to the forgotten age. Is there a particular reason to remove this?